app/controllers/api/v1/users_controller.rb

```diff
@@ -1,7 +1,18 @@
 class Api::V1::UsersController < Api::V1::BaseController
-
   def logged
     @user = current_user
   end
 
+  def invite_to_collab
+    user = User.find(params[:id])
+    skills = SharedSkillsMatcher.new(user, current_user).call
+    return 'No shared skills' if skills.blank?
+
+    CollabMailer.with(
+      from_user: current_user,
+      to_user: user,
+      skills: skills
+    ).invite_to_collaborate.deliver_now
+    render json: { message: "Invitation to #{user.username} sent" }, status: 200
```

Comment:
I wonder if I could use a sweet alert for this on the front end 🤔

Reply:
if it were standard rails it'll be easy, here you need to make your own alert with the json message :)

Reply:
https://www.npmjs.com/package/sweetalert2-react
Should be straight forward.